KNOWN_ISSUES.md

```diff
@@ -1,10 +1,24 @@
-# Frequent issues in latest stable release
+# Frequent issues in latest releases
 
 ## [Unable to mirror images from quay](https://github.com/openshift/okd/issues/402)
-  This blocks air-gapped installs, the fix needs to land in our buildfarms and all images rebuild.
+  **Effected Versions:** All  
+  **Description:**  Attempts to mirror content on the quay repository fails. This blocks air-gapped installs on restricted networks. The fix needs to land in our buildfarms and all images rebuilt.  
+  **Workaround**: None available at this time. 
 
-## [Invalid signature store used](https://github.com/openshift/okd/issues/426)
-  Upgrades to latest stable version require `--force` when `oc adm upgrade` call
+## [Invalid signature error when attempting cluster updates](https://github.com/openshift/okd/issues/426)
+  **Effected Versions:** 4.6.0-0.okd-2020-11-27-200126, 4.6.0-0.okd-2020-12-12-135354, 4.6.0-0.okd-2021-01-17-185703  
+  **Description:** Cluster upgrades will fail due to incorrect image signatures.   
+  **Workaround:** Perform the upgrade in the terminal using the --force flag  
+  ```bash
+  oc adm upgrade --to-latest=true --force=true     
```

Comment:
If you already tried to upgrade above command won't be enough.
Output will be like:
```
oc adm upgrade --to-latest=true --force=true
error: already upgrading.

  Reason: ImageVerificationFailed
  Message: Unable to apply 4.6.0-0.okd-2021-01-23-132511: the image may not be safe to use



If you want to upgrade anyway, use --allow-upgrade-with-warnings.
```

Reply:
And then:

```
# oc adm upgrade --to-latest=true --force=true --allow-upgrade-with-warnings
warning: --allow-upgrade-with-warnings is bypassing: already upgrading.

  Reason: ImageVerificationFailed
  Message: Unable to apply 4.6.0-0.okd-2021-01-23-132511: the image may not be safe to use

warning: --force overrides cluster verification of your supplied release image and waives any update precondition failures.
Updating to latest version 4.6.0-0.okd-2021-01-23-132511
```

Reply:
Right, we should add `oc adm upgrade --reset && oc adm update ..` instead

Reply:
Added --reset before forced upgrade and use of --allow-upgrade-with-warnings for forced upgrade.